packages/authorize/useAuthorize.js

```diff
@@ -116,5 +116,5 @@ export default (
     // eslint-disable-next-line react-hooks/exhaustive-deps
   }, [organizationId, region, customerId, permissions]);
 
-  return [authorized, loading];
+  return [authorized, loading, currentRegion];
```

Comment:
need to update TS def: https://github.com/Availity/availity-react/blob/master/packages/authorize/useAuthorize.d.ts#L13